rtl/eclrtl/rtlkey.cpp

```diff
@@ -1860,7 +1860,7 @@ void CStringSet::addRange(const void *loval, const void *hival)
         int rc;
         while ((int)a<b)
         {
-            int i = (a+b+1)/2;
+            int i = a+(b+1-a)/2;
```

Comment:
Maybe.  The rules on overflow for unsigned types are well defined, so I they will be equivalent.